apps/publications/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,30 @@
+<mat-toolbar [ngStyle]="{'background': bgColor}" class="header-container" style="display: flex; justify-content: space-between;">
+  <div style=" display: flex; flex-direction: row;">
+    <button (click)="sideNav.toggle()" [ngStyle]="{'color': iconColor}" mat-icon-button class="mt-auto mb-auto">
+      <mat-icon>menu</mat-icon>
+    </button>
+
+    <a class="ml-3" [innerHTML]="logo" routerLink="/all-publications"></a>
+    <mat-icon svgIcon="perun-publications-white" style="height: 3rem; width: 3rem; cursor: pointer; outline: 0" class="mt-auto mb-auto ml-2" routerLink="/all-publications"></mat-icon>
```

Comment:
please never use "style", always when to style some element do it in scss file
![image](https://user-images.githubusercontent.com/32871167/110105280-7a9cb480-7da8-11eb-9c69-1e01c92ad6a2.png)
please also make these two icons a little bit smaller like it is shown in picture, also make them look centered and fix the gaps between each other(the gaps on the pictures are wild)